app/models/agent.rb

```diff
@@ -19,8 +19,8 @@ class Agent < ActiveRecord::Base
 
   load_types_in "Agents"
 
-  SCHEDULES = %w[every_1m every_2m every_5m every_10m every_30m every_1h every_2h every_5h every_12h every_1d every_2d every_7d
-                 midnight 1am 2am 3am 4am 5am 6am 7am 8am 9am 10am 11am noon 1pm 2pm 3pm 4pm 5pm 6pm 7pm 8pm 9pm 10pm 11pm never]
+  SCHEDULES = %w[every_1m every_2m every_5m every_10m every_30m every_1h every_2h every_5h every_12h every_1d every_2d] + [OpenStruct.new(to_s: 'every_7d', humanize: 'Every Monday')] +
```

Comment:
Where does `humanize` get called? I guess this won't require a migration for Agents using every_7d because the db string won't change?